tests/test_api.py

```diff
@@ -141,6 +151,30 @@ def test_relations(self, client, entities, resource, field):
             assert type(body[field]) is str
             assert len(body[field]) == 11
 
+    @pytest.mark.parametrize('method', ['POST', 'PATCH'])
+    @pytest.mark.parametrize('value', [12000,
+                                       '12000',
+                                       'hai der'])
+    @pytest.mark.parametrize('endpoint, field',
+                             [('/aliquots', 'shipment_date')])
+    def test_custom_datetime_field(self, client, entities, endpoint, method,
```

Comment:
May be worth turning this into a general test_bad_input check that takes a method, endpoint, field, and value and sends that.

Reply:
Oo good idea 👍